eth/backend.go

```diff
@@ -63,6 +63,9 @@ var (
 )
 
 type Config struct {
+	ChainConfig *core.ChainConfig // chain configuration
+
+	Testnet   bool   // Indicates whether we're serving the testnet
```

Comment:
No, it's actually **THE** chain config. All loading now happens in package `util`.